lib/src/pubspec.dart

```diff
@@ -23,6 +23,12 @@ import 'utils.dart';
 final _packageName = new RegExp(
     "^${identifierRegExp.pattern}(\\.${identifierRegExp.pattern})*\$");
 
+/// The default SDK constraint for packages that don't declare one.
+///
+/// This allows 2.0.0 dev versions to make the migration proecss smoother.
+final _defaultSdkConstraint =
+    new VersionConstraint.parse("<2.0.0-dev.infinity");
```

Comment:
Perhaps this is a little *too* forward looking, but is it worth making this automatically be "<X.0.0-dev.infinity" where X is the current major version of the SDK?